src/main/java/net/atos/zac/util/LocalDateUtil.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.util;
+
+import java.time.LocalDate;
+
+public final class LocalDateUtil {
+
+    private LocalDateUtil() {
+    }
+
+    public static boolean dateNowIsBetweenInclusive(LocalDate begin, LocalDate end) {
+        final LocalDate now = LocalDate.now();
+        return (begin == null || begin.isBefore(now) || begin.isEqual(now)) && (end == null || end.isAfter(now) || end.isEqual(now));
```

Comment:
Ik denk dat begin nooit null is toch? Op dit soort geldigheidsheids-datum range is gewoonlijk alleen de einddatum optioneel.

En verder is de controle die je nodig hebt dan:

begin <= vandaag && ( eind==null || vandaag < eind )

Dus inclusief begin en exclusief eind. Dat heeft alles te maken met dat elke datum/tijd (ongeacht de resolutie) altijd een floor is. Als je dat consequent doet heb je nooit gezeur met randgevallen.

Reply:
Ik houd de mogelijkheid voor `begin == null` wel in de methode, daarmee is het gedrag van de util-methode beter voorspelbaar. De logica op de einddatum is aangepast in de opvolgende commit.